R/engine_multiverse.R

```diff
@@ -130,16 +130,11 @@ multiverse_default_block_exec <- function(.code, options, knit = FALSE) {
   } else {
     # when in interactive mode, execute the default analysis in the knitr global environment
     
-    # first and last elements of `code` are "{" and "}" so we have to strip them.
-    # (otherwise only the last thing would be printed as the entire expression would
-    # only return one object)
-    code = .code[-c(1, length(.code))]
```

Comment:
When I ``rmarkdown::render("vignettes/frequentist-multiverse-analysis.Rmd")`, `{` and `}` are not present.

Reply:
Where do the `{`/`}` come from? Are you adding them to the code somewhere in multiverse?

Reply:
(sorry for the delayed response) yes! I don't recall why we had to implement it this way, but we needed to have the entire code from a code block enclosed in `{` / `}` to get it to work 
https://github.com/MUCollective/multiverse/blob/afd31e391a6b207accad5b248344d43552df5a8b/R/inside.R#L107